JenkinsFile

```diff
@@ -61,9 +61,11 @@ node {
             cf set-env ${fullAppName} SPACE ${pcfSpace}
             cf set-env ${fullAppName} DOMAIN ${pcfDomain}
             cf set-env ${fullAppName} PIAZZA_API_KEY ${piazzaApiKey}
-            cf set-env ${fullAppName} GEOAXIS ${env.GEOAXIS_DOMAIN}
-            cf set-env ${fullAppName} GEOAXIS_CLIENT_ID ${GEOAXIS_CLIENT_ID}
-            cf set-env ${fullAppName} GEOAXIS_SECRET ${GEOAXIS_SECRET}
+            cf set-env ${fullAppName} OAUTH_AUTH_DOMAIN ${env.GEOAXIS_AUTH}
```

Comment:
John, can you please Crosscheck these with the https://github.com/venicegeo/jenkins/blob/master/venice.json seed job? 

GEOAXIS_AUTH doesnt exist. You mean GEOAXIS_AUTH_DOMAIN?  GEOAXIS doesn't exist.  GEOAXIS_LOGOUT doesnt exist (but GEOAXIS_LOGOUT_DOMAIN does).

Reply:
ah yep, you're absolutely right.

Reply:
Pushed a fix for this and also fixed JenkinsFile.promote